File: FATEC/Arquitetura e organização de computadores/C/lista 1/ex1.c
#include <stdio.h>
int main()  

{

    int quant_antes,quant_depois,quant_mais;

    printf("Qual e a quantidade atual desse produto no estoque?");
    scanf("%d",&quant_antes);

    printf("Qual e a quantidade de unidades que vai chegar desse produto?");
    scanf("%d",&quant_mais);

    quant_depois = quant_antes + quant_mais;

    printf("A nova quantidade sera de %d unidades.", quant_depois);

return 0;

}

File: FATEC/algaritmo e lógica/python/lista 1/ex1.py
quant_antes = int(input("Qual é a quantidade desse produto já no estoque?"))
quant_mais = int(input("Qual é a quantidade de unidades desse produto que irá chegar?"))

quant_depois = quant_antes + quant_mais

print(f"A nova quantidade desse produto no estoque será de {quant_depois} unidades")